composer.json

```diff
@@ -304,7 +304,7 @@
             "vendor/bin/changelog-linker cleanup --ansi"
         ],
         "docs": [
-            "bin/rector dump-rectors > docs/rector_rules_overview.md",
+            "bin/rector dump-rectors --output-file docs/rector_rules_overview.md --ansi",
```

Comment:
@samsonasik This is the new updated version to generated rules overview, btw

Reply:
thank you 👍